Glossary_of_Terms.txt

```diff
@@ -0,0 +1,37 @@
+Here are a list of some proper nouns and terms which are different in Compound X then in the English Language normally. If one is missing, please create an issue in Github asking for it! Hope it helps!
+
+Action: In Compound X, an Action rarely means a single abstract bodily movement as it does in English. Instead an Action is a measure of time specifically during a Player's turn in Combat. See Actions in Basic Rules.txt
```

Comment:
is "Combat" going to be an entry? If not it shouldn't be capitalized.

Reply:
Added